monitor/io.py

```diff
@@ -0,0 +1,43 @@
+import os
+import subprocess
+import numpy as np
+from tempfile import mkstemp
+from shutil import move, rmtree
+
+######### ----------------------------------------###########
+
+def proc_subprocess(executing_arguments, dir):
+    proc = subprocess.Popen( ' '.join(executing_arguments),
+                            shell=True,
+                            stderr=subprocess.PIPE,
+                            stdout=subprocess.PIPE )
+    retvals = proc.communicate()
+
+    stdout = retvals[0]
+    stderr = retvals[1]
+    returncode = proc.returncode
+
+    with open(dir+'log_file', "a" ) as logfile:
```

Comment:
use `os.path.join` to create path here.

Also, I'd suggest adding a `.txt` suffix to your log file name so that it clear that your log_file is readable by a text editor.